components/footer/index.scss

```
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) IBM, Corp. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+@import "@carbon/colors/scss/colors";
+@import "@carbon/layout/scss/layout";
+@import "@carbon/type/scss/type";
+
+:host {
+  margin: 0;
+}
+
+footer {
+  display: block;
+}
+
+.secondary-footer {
+  display: block;
+  padding: 2rem 0 4.5rem;
+}
+
+.main-footer {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 2rem 0;
+  padding-right: 2.5rem;
+  padding-left: 2.5rem;
+  background-color: #f2f4f8;
+}
+
+.app-logo {
+  flex: 1.5 150px;
+  flex-basis: 0;
+  margin-right: auto;
+  font-family: carbon--font-family("sans");
+  fill: #697077;
+
+  svg {
+    height: 32px;
+  }
+
+  path {
+    background-color: #697077;
+    color: #697077;
+    fill: #697077;
+  }
+}
+
+.footer-section-title {
+  display: block;
+  margin-bottom: 1.5rem;
+  color: #697077;
+  font-weight: 600;
+  font-size: 0.875rem;
+  font-family: "IBM Plex Sans SemiBold", sans-serif;
+  line-height: 1.25;
+  letter-spacing: 0.16px;
+}
+
+li {
+  list-style: none;
+}
+
+ul {
+  padding-left: 0;
+}
+
+.link {
+  display: inline-block;
+  width: 100%;
+  padding-bottom: 0.5rem;
+  color: #697077;
+  font-weight: 400;
+  font-size: 0.75rem;
+  line-height: 1.3333;
+  letter-spacing: 0.32px;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.social-media {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: auto 1fr;
+  grid-gap: 0 1rem;
```

The original had more vertical spacing, it was a matter of padding, but I think we can safely add it here.
Also we should use [spacing variables](https://carbondesignsystem.com/guidelines/spacing/overview/) from carbon.

```suggestion
  grid-gap: $spacing-03 $spacing-05;
```

---

I added an additional 0.5 rem of padding bottom